### PR TITLE
Update Dockerfile to 0.2.0

### PR DESCRIPTION
**Pull request by RobAtticus**

### CODE REVIEW

File: postgis/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM timescale/timescaledb:0.0.11-beta
+FROM timescale/timescaledb:0.2.0
```
Comment (cevian) [inline postgis/Dockerfile]: I think this is inconsistent with the commit message

File: postgis/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM timescale/timescaledb:0.0.11-beta
+FROM timescale/timescaledb:latest
```
Comment (erimatnor) [inline postgis/Dockerfile]: This is a bit weird since it might not match the ENV variable in the Docker file, leading to inconsistent builds.

Comment (erimatnor) [inline postgis/Dockerfile]: Another issue is that you typically have a latest tag cached in your local Docker. If you want to use latest tag you should make sure to always fetch the newest "latest" tag first.

Comment (RobAtticus) [inline postgis/Dockerfile]: Okay, will revert back to number

**Pull request merged**